src/app/static/src/tests/components/projects/shareProject.test.ts

```diff
@@ -628,4 +628,91 @@ describe("ShareProject", () => {
         expect(mockTooltip.mock.calls[0][1].value).toBe("Partager");
     });
 
+    it("if email entered is valid, cycles to empty input then ok button on enter presses", async (done) => {
+        const cloneProject = jest.fn();
+        const wrapper = mount(ShareProject, {
+            attachToDocument: true,
```

Comment:
This is deprecated but typescript doesn't recognise attachTo as a type on mount.

Reply:
Are we aware of any potential solutions to this? Updating type definitions?

Reply:
I updated the type definitions to include attachTo and tried with it but it didn't work with it either. I think the version of Vue tests we have doesn't recognise attachTo.

Reply:
Ah - yes. It wasn't introduced until the [final 1.0.0 release](https://github.com/vuejs/vue-test-utils/releases/tag/v1.0.0). Suggest we update via another ticket - I just tried it and it causes lots of test failures.